triggers/service/auth/src/test/scala/com/daml/auth/middleware/oauth2/CliSpec.scala

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.auth.middleware.oauth2
+
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AsyncWordSpec
+import com.daml.bazeltools.BazelRunfiles.requiredResource
+
+object CliSpec {
+  def resourceFile(name: String) = {
+    getClass.getClassLoader.getResource(name)
+  }
+
+}
+class CliSpec extends AsyncWordSpec with Matchers {
```

Comment:
It's quite important that we nail how the configuration can be alternatively specified and/or overridden using environment variables. This should be handled natively by Lightbend Config, but can you add a couple of tests in that regard?

Reply:
Let me look into this properly , attempts to find ways to pass env variables via `env` property into our `da_scala_test` rules were futile yesterday.